src/Views/PartitioningView.vala

```diff
@@ -190,34 +192,38 @@ public class Installer.PartitioningView : AbstractInstallerView {
     private void validate_status () {
         uint8 flags = 0;
         const uint8 ROOT = 1;
-        const uint8 BOOT = 2;
+        const uint8 EFI = 2;
```

Comment:
It's really time to create an:

[flags]
public enum Status {
    ROOT,
    EFI
}